rpm/spec.in

```diff
@@ -49,7 +53,7 @@ fi
 %install
 %{__rm} -rf %{buildroot}
 %{__make} install  DESTDIR="%{buildroot}" AM_INSTALL_PROGRAM_FLAGS=""
-mkdir -p $RPM_BUILD_ROOT/
+%{__install} -m 0644 wolfssl/options.h %{buildroot}/usr/include/wolfssl/options.h
```

Comment:
Why is this needed now?

Reply:
```
mkdir -p $RPM_BUILD_ROOT/
```
Use less here and can be removed

Reply:
Fixed https://github.com/wolfSSL/wolfssl/pull/7086